setup.py

```diff
@@ -1,10 +1,23 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
+import os
+import sys
 import versioneer
 
 README = open('README.rst', 'r').read()
 
 
+install_requires = [
+    'future>=0.14.0; python_version<"3"',
+    'futures; python_version<"3.2"',
+]
+
+if sys.version_info[0] == 2:
+    if os.name != 'nt':
+        install_requires.append('ujson<=2.0.3')
```

Comment:
```suggestion
    install_requires.append('ujson<=2.0.3; platform_system!="Windows"')
```

That previous version was the right way to declare that a dependency doesn't apply for Windows.